gpkit/solution_array.py

```diff
@@ -231,14 +231,18 @@ def results_table(data, title, minval=0, printunits=True, fixedcols=True,
             lines.append([varstr, valstr, units, label])
         else:
             varstr = "$%s$" % varstr.replace(" : ", "")
+            boxunit = var.unitstr()
```

Comment:
also there's the `into` arg; square brackets with `[-]` for dimensionless quantities could be as simple as a couple of arguments that get passed to unitstr